catalog/templates/base_generic.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  {% block title %}<title>{% translate "Local Library" %}</title>{% endblock %}
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+  <!-- Add additional CSS in static file -->
+  {% load static %}
+  <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+</head>
+<body>
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-sm-2">
+      {% block sidebar %}
+        <ul class="sidebar-nav">
+          <li><a href="{% url 'index' %}">{% translate "Home" %}</a></li>
```

Comment:
dùng `trans` là đc nha em
sửa tương tự
```suggestion
          <li><a href="{% url 'index' %}">{% trans "Home" %}</a></li>
```

Reply:
Em thấy trên docs họ ghi thế này:
`Changed in Django 3.1:
The trans tag was renamed to translate. The trans tag is still supported as an alias for backwards compatibility.`
Và họ hướng dẫn cũng dùng translate nên em để translate tất ạ.